packages/react-aria-components/src/FileTrigger.tsx

```diff
@@ -33,14 +33,14 @@ export interface FileTriggerProps extends SlotProps, DOMProps, AriaLabelingProps
   /**
    * Handler when a user selects a file.
    */
-  onChange?: React.ChangeEventHandler<HTMLInputElement>
+  onChange?: (value: FileList) => void
```

Comment:
nit, I think we probably want to call it `files` here
```suggestion
  onChange?: (files: FileList) => void
```